pkg/client/client.go

```diff
@@ -423,28 +423,30 @@ func (c *Client) Fetch(ctx context.Context, request FetchRequest) (res *FetchRes
 
 	for _, providerConfig := range request.Providers {
 		providerConfig := providerConfig
+		createdAt := time.Now().UTC()
+		fs := FetchSummary{
+			FetchId:       fetchId,
```

Comment:
Maybe also save CloudQuery core version?

Reply:
that is a good suggestion